libraries/AP_HAL_Linux/SPIDevice.cpp

```diff
@@ -107,6 +107,12 @@ SPIDesc SPIDeviceManager::_device[] = {
     SPIDesc("lsm9ds0_g",  0, 0, SPI_MODE_3, 8, RPI_GPIO_12,  10*MHZ, 10*MHZ),
     SPIDesc("raspio",     0, 0, SPI_MODE_3, 8, RPI_GPIO_7,   10*MHZ, 10*MHZ),
 };
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_OCPOC_ZYNQ
+SPIDesc SPIDeviceManager::_device[] = {
+    /* MPU9250 is restricted to 1MHz for non-data and interrupt registers */
+    SPIDesc("mpu9250",    1, 0,  SPI_MODE_3, 8, SPI_CS_KERNEL,  1*MHZ, 1*MHZ),
```

Comment:
is 1MHz a limitation on your bus?  It should usually go to 10MHz on mpu9250 after the initialization.  No problem if it is, just want to double check this is intended.

Reply:
Thanks for catching this, I'm not sure why it was at 1MHz as we go to 10MHz in PX4. I've changed it accordingly.